recipes/sos-bioinfo/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sos-bioinfo" %}
+{% set version = "0.9.10.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "47fdd8399512a4a7f98e97225f4811c6aa73787cad8713c513a3e9cc65417f63" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [not py36]
+  script: python -m pip install --no-deps --ignore-installed . --verbose
+
+requirements:
+  host:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```

Comment:
Is setuptools really a runtime dependency?

Reply:
no.